spec/services/persist_basic_contained_output_file_service_spec.rb

```diff
@@ -13,13 +13,16 @@ class BasicContainerObject < ActiveFedora::Base
   let(:file)              { File.new(file_path)}
   let(:destination_name)  { 'the_derivative_name' }
 
-  context "when file is basic contained (default assumption)" do  # alas, we have to support this as the default because all legacy code (and fedora 3 systems) created basic contained files
```

Comment:
Try this:
```
  let(:object)            { BasicContainerObject.new  }
  let(:file_path)         { File.join(fixture_path, 'test.tif') }
  let(:file)              { Hydra::Derivatives::IoDecorator.new(File.new(file_path))}
  let(:destination_name)  { 'the_derivative_name' }

  context "when file is basic contained (default assumption)" do  # alas, we have to support this as the default because all legacy code (and fedora 3 systems) created basic contained files
    it "persists the file to the specified destination on the given object" do
      file.original_name = destination_name
      described_class.call(object, file)
      expect(object.send(destination_name.to_sym).content).not_to be_nil
      expect(object.send(destination_name.to_sym).content_changed?).to eq false
    end
  end
```